apps/prettier-git/src/prettier-git.ts

```diff
@@ -0,0 +1,448 @@
+import * as child_process from 'child_process';
+import { once } from 'events';
+import * as path from 'path';
+import { StringDecoder } from 'string_decoder';
+
+import * as prettier from 'prettier';
+
+import type { IPackageJson } from '@rushstack/node-core-library';
```

Comment:
Maybe explicitly mention near the imports that only `prettier` and system modules are allowed to be imported into this project at runtime.